openapi.yaml

```diff
@@ -3,6 +3,9 @@ info:
   title: Rekor
   description: Rekor is a cryptographically secure, immutable transparency log for signed software releases.
   version: 0.0.1
+servers:
```

Comment:
I think you'll need to use `host` and `schemes` instead which is the v2 way of expressing this.

Reply:
that fixed it, thanks bob